src/napi/node_api_function.c

```diff
@@ -0,0 +1,89 @@
+/* Copyright 2018-present Rokid Co., Ltd. and other contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "jerryscript-ext/handle-scope.h"
+#include "jerryscript.h"
+#include <stdlib.h>
+#include "internal/node_api_internal.h"
+#include "node_api.h"
+
+static const jerry_object_native_info_t native_obj_type_info = { .free_cb =
+                                                                     free };
+
+static jerry_value_t handler(const jerry_value_t function_obj,
```

Comment:
This function name seems too common, clang are not like JavaScript, all symbols will be written without scope, it's also not recommended like this, just add a prefix is better :)